test/server/cards/03_TWI/events/HelloThere.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe('Hello There', function() {
+    integration(function(contextRef) {
+        describe('Hello There\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['hello-there', 'green-squadron-awing'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        hand: ['atst'],
+                        groundArena: ['consular-security-force'],
+                    }
+                });
+            });
+
+            it('should apply -4/-4 to a unit which entered play this phase', function () {
+                const { context } = contextRef;
+
+                // play a-wing & opponent plays atst
+                context.player1.clickCard(context.greenSquadronAwing);
+                context.player2.clickCard(context.atst);
+
+                // play hello there, should be able to select a-wing & at-st
```

Comment:
We can't use setupTest to test things that enter play because the test setup doesn't actually put units into play according to the game rules, it cheats them in. So we need to explicitly have a previous phase where we play out some units to correctly test anything using this watcher.